File: TransactionSubsystem/Entities/Transaction.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TransactionSubsystem.Entities
{
    public class Transaction
    {
        public int Id { get; set; }

        [NotMapped]
        public bool CommitAvailableState { get; set; }

        public decimal Amount { get; set; }
        public User TransactionOwner { get; set; }
        public User Recepient { get; set; }
        public DateTime Date { get; set; }
        public decimal ResultingBalance { get; set; }
    }
}

File: ParrotWings/app/core/domain/transaction.ts
export class Transaction {
    Amount: number;
    CorrespondedUser: string;
    Date: Date;
    ResultingBalance: number;
    AmountValid: boolean;
    CorrespondedUserValid: boolean;

    constructor(        
		//amount: number,
		//recepientName: string,
		//date: Date,
		//resultingBalance: number
		) 
    {
        this.AmountValid = true;
        this.CorrespondedUserValid = true;
        //this.Amount = amount;
        //this.RecepientName = recepientName;
		//this.Date = date;
        //this.ResultingBalance = resultingBalance;
    }
}